common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java

```diff
@@ -1010,4 +1010,18 @@ public static Map convertJsonStringToMap(String jsonString) throws IOException {
     map.put(ENDDATE, new SimpleDateFormat(YYYY_MM_DD_FORMATTER).format(cal.getTime()));
     return map;
   }
+  
+  /**
+  * This method util will be used to create ProjectCommonException for all kind
+  * of client error for the given response code(enum).
+  *
+  * @param ResponseCode
```

Comment:
define ResposneCode :  An enum of all the api responses.

Reply:
done